### PR TITLE
FAQ: Can the party that calls SPC validate the assertion even if not the RP?

### PR DESCRIPTION
**Pull request by ianbjacobs**
Based on conversations about SPC I thought it might be useful to start an FAQ.

I started it with a first question: 
 Can the party that calls SPC validate the assertion even if not the RP?

Ian

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -288,6 +288,11 @@ The authenticator signature is emphemeral to each transaction and is not useful
 [Web Authentication]: https://www.w3.org/TR/webauthn/
 [pilot]: https://bit.ly/webauthn-to-pay-2020h2-pilot
 
+## FAQ
+
+### Q. Can the party that calls SPC validate the assertion even if not the RP?
```
Comment (danyao) [inline README.md:293]:
```suggestion
### Q. Who can validate the SPC assertion besides the actual Relying Party (RP)?
```

File: README.md [hunk 2]
```diff
@@ -288,6 +288,11 @@ The authenticator signature is emphemeral to each transaction and is not useful
...
+
+Yes. The Relying Party can share the credentialPublicKey (of the authenticator data) with another party, e.g., through an out-of-band communication.
```
Comment (danyao) [inline README.md:295]:
```suggestion
Validation in the context of SPC means the ability to verify that the signature in the SPC response is a signature over the SPC challenge (which bundles together transaction details with transaction-specific dynamic data from the RP). An RP can choose to share the credential public key with another party (e.g. the card network or PSP) via out-of-band communication to enable said party to validate the SPC assertion.
```

**Review by danyao — COMMENTED** [review]
@ianbjacobs Tweaked the wording a bit. WDYT?

**Comment by ianbjacobs** [comment]
@danyao,
Thanks for the suggestions. I took your text, moved the bits around, and updated the proposal.

**Pull request merged**